test/utility/backup.c

```diff
@@ -84,17 +93,16 @@ testutil_backup_create_full(WT_CONNECTION *conn, const char *home_dir, const cha
  *     parameters.
  */
```

Comment:
Add a comment to identify the directories used such as the below.
`*   The directory containing the source is BACKUP_BASE.source_id.
*     The new incremental backup directory is BACKUP_BASE.backup_id.`